core/src/main/java/org/apache/carbondata/core/carbon/querystatistics/QueryStatisticsRecorder.java

```diff
@@ -61,14 +67,275 @@ public QueryStatisticsRecorder(String queryId) {
    */
   public synchronized void recordStatistics(QueryStatistic statistic) {
     queryStatistics.add(statistic);
+    // refresh query Statistics Map
+    String key = statistic.getQueryId();
+    if (!StringUtils.isEmpty(key)) {
+      // 240954528274124_0 and 240954528274124 is the same query id
+      key = key.substring(0, 15);
+    }
+    if (queryStatisticsMap.get(key) != null) {
+      queryStatisticsMap.get(key).add(statistic);
+    } else {
+      List<QueryStatistic> newQueryStatistics = new ArrayList<QueryStatistic>();
+      newQueryStatistics.add(statistic);
+      queryStatisticsMap.put(key, newQueryStatistics);
+    }
   }
 
   /**
    * Below method will be used to log the statistic
    */
   public void logStatistics() {
     for (QueryStatistic statistic : queryStatistics) {
-      LOGGER.statistic(statistic.getStatistics(queryIWthTask));
+      LOGGER.statistic(statistic.getStatistics());
+    }
+  }
+
+  /**
+   * Below method will be used to show statistic log as table
+   */
+  public void logStatisticsTable() {
+    String tableInfo = putStatisticsIntoTable();
+    if (null != tableInfo) {
+      LOGGER.statistic(tableInfo);
+    }
+  }
+
+  /**
+   * Below method will parse queryStatisticsMap and put time into table
+   */
+  public String putStatisticsIntoTable() {
+    for (String key: queryStatisticsMap.keySet()) {
+      try {
+        // TODO: get the finished query, and print Statistics
+        if (queryStatisticsMap.get(key).size() > 8) {
+          String jdbc_connection_time = "";
+          String sql_parse_time = "";
+          String load_meta_time = "";
+          String block_identification_time = "";
+          String schedule_time = "";
+          String driver_part_time = "";
+          String executor_part_time = "";
+          String load_index_time = "";
+          String scan_data_time = "";
+          String dictionary_load_time = "";
```

Comment:
dictionary loading time required in both the side executor and driver, please add in driver side